backend/users/autocomplete.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+from django.contrib.admin.widgets import SELECT2_TRANSLATIONS, AutocompleteSelect
+from django.urls import reverse
+from django.utils.translation import get_language
+
+from users.client import get_users_data_by_ids
+
+
+class UsersBackendAutocomplete(AutocompleteSelect):
+    def __init__(self, admin_site, attrs=None, choices=(), using=None):
+        self.admin_site = admin_site
+        self.db = using
+        self.choices = choices
+        self.attrs = {} if attrs is None else attrs.copy()
+        self.i18n_name = SELECT2_TRANSLATIONS.get(get_language())
```

Comment:
does this init do anything custom?

Reply:
the default init wants the field as input

Reply:
see the base code here: https://github.com/django/django/blob/8eb56930915f8022aa27ec29ab5be7d21764608c/django/contrib/admin/widgets.py#L385